src/main/java/Pac.java

```diff
@@ -0,0 +1,229 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Pac {
+    static String logo = " ____     ___    _____\n"
+            + "|  _ \\   / _ \\  |  ___|\n"
+            + "| |_| | | |_| | | |\n"
+            + "|  __/  | | | | | |___\n"
+            + "|_|     |_| |_| |_____|\n";
+    static String newline = "----------------------------------------------------";
+    static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        System.out.println("\n" + newline + "\n" + newline + "\n" + logo);
+        System.out.println("Hello there! I'm Pac, your very own Personal Assistant ChatBot.\nHow may I help you?");
+        System.out.println(newline + "\n");
+
+        Scanner sc = new Scanner(System.in);
+
+        while (true) {
+            String input = sc.nextLine();
+            try {
+                String[] inputArray = input.split(" ", 2);
+                String firstword = inputArray[0].toLowerCase();
+                Keyword keyword;
+
+                switch (firstword) {
+                    case "bye":
+                        keyword = Keyword.BYE;
+                        break;
+                    case "list":
+                        keyword = Keyword.LIST;
+                        break;
+                    case "mark":
+                        keyword = Keyword.MARK;
+                        break;
+                    case "unmark":
+                        keyword = Keyword.UNMARK;
+                        break;
+                    case "todo":
+                        keyword = Keyword.TODO;
+                        break;
+                    case "deadline":
+                        keyword = Keyword.DEADLINE;
+                        break;
+                    case "event":
+                        keyword = Keyword.EVENT;
+                        break;
+                    case "delete":
+                        keyword = Keyword.DELETE;
+                        break;
+                    default:
+                        keyword = Keyword.ERROR;
+                        break;
+                }
```

Comment:
can consider using a hashmap to lookup the right Keyword given a string? same functionality but may save some lines, may look slightly more readable as well, but it's up to you